roles/postgres/tasks/main.yml

```diff
@@ -56,3 +56,26 @@
   with_items: "{{ postgres_users }}"
   become: true
   become_user: postgres
+
+- name: Add a service template that allows creating backups of postgresql databases
+  ansible.builtin.template:
+    src: postgres-backups@.service.j2
+    dest: /usr/lib/systemd/system/postgres-backups@.service
+  become: true
+
+- name: Add timers triggering backups of postgresql databases
+  ansible.builtin.template:
+    src: postgres-backups@.timer.j2
+    dest: /usr/lib/systemd/system/postgres-backups@{{ item.database }}.timer
+  with_items: "{{ postgres_users }}"
+  when: item.backup_schedule is defined and item.backup_schedule
```

Comment:
nit: I may be wrong, but I think we can use `item.backup_schedule is truthy`. I don't mind keeping it this way though.

Reply:
It's not clear to me from the docs how undefined  values are handled: https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#testing-truthiness :(